common/src/test/scala/weco/storage_service/storage/services/s3/S3UploaderTest.scala

```diff
@@ -1,170 +0,0 @@
-package weco.storage_service.storage.services.s3
```

Comment:
This seems to be duplicated across repos – see also https://github.com/wellcomecollection/scala-libs/blob/3397afbb8624096d26f8d91b64acbc5fb0b54027/storage/src/test/scala/weco/storage/services/s3/S3UploaderTest.scala#L17

It probably should live in this repo, but for now deleting it to avoid duplication of effort.